### DISCUSSION TITLE
[DEPRECATING] v1.1-next1: store --cache flag deprecated since 1.0.x to be removed

### DISCUSSION THREAD

**Discussion by dweomer** [hauler-dev/hauler#359] [Deprecations]
This will mean that continued usage of the `hauler store --cache` flag will beget "unknown flag" errors.

See:
- https://github.com/hauler-dev/hauler/pull/355
- [milestone v1.1-next1](https://github.com/hauler-dev/hauler/milestone/11)

**Comment by zackbradys** [comment]
Closing as resolved due to https://github.com/hauler-dev/hauler/releases/tag/v1.1.1
